doc/man3/SSL_shutdown.pod

```diff
@@ -260,6 +260,15 @@ These functions do not affect an underlying network BIO or the resource it
 represents; for example, a UDP datagram provided to a QUIC connection as the
 network BIO will remain open.
 
+Note that when using QUIC, an application must call SSL_shutdown() if it wants
+to ensure that all transmitted data was received by the peer. This is unlike a
+TLS/TCP connection, where reliable transmission of buffered data is the
+responsibility of the operating system. If an application calls SSL_free() or
```

Comment:
`... calls SSL_free() on the QUIC connection SSL object ...` as calling SSL_free() on the stream object is fine.